primedev/engine/runframe.cpp

```diff
@@ -3,17 +3,14 @@
 #include "hoststate.h"
 #include "server/serverpresence.h"
 
-AUTOHOOK_INIT()
-
-// clang-format off
-AUTOHOOK(CEngine__Frame, engine.dll + 0x1C8650,
-void, __fastcall, (CEngine* self))
-// clang-format on
+static void(__fastcall* o_pCEngine__Frame)(CEngine* self) = nullptr;
+static void __fastcall h_CEngine__Frame(CEngine* self)
 {
-	CEngine__Frame(self);
+	o_pCEngine__Frame(self);
```

Comment:
Does this appear in the logs?
The old hook did (`Enabling hook CEngine__Frame`), but I can't find any trace of the new one.

Reply:
@ASpoonPlaysGames

Reply:
non-auto hooks dont log when they are enabled iirc